app/agents/voice/automatic/tools/breeze/utils.py

```
@@ -230,14 +230,19 @@ def remove_html_tags(html_text: str) -> str:
 
 
 # Surcharge-specific utility functions
-def detect_surcharge_rule_overlaps(new_rules, existing_rules, payment_type):
+def detect_surcharge_rule_overlaps(
+    new_rules, existing_rules, payment_type, payment_method_type, payment_method
```

Ruff (0.13.2)

Unused function argument: `payment_method`

(ARG001)